src/mobility/queries/use-geofencing-zones.tsx

```diff
@@ -0,0 +1,20 @@
+import {useQuery} from '@tanstack/react-query';
+import {getGeofencingZones} from '@atb/api/mobility';
+import {useGeofencingZonesEnabled} from '../use-geofencing-zones-enabled';
+
+const ONE_MINUTE = 1000 * 60;
+
+export const useGeofencingZonesQuery = (systemIds: string[]) => {
```

Comment:
Is there ever a use case for multiple system ids? I can't wrap my head around situations where we present geofencing zones for multiple providers at the same time, without it being messy for the user.

Reply:
Not really, this was only used because the mobility api takes in an array. We can change it to just `systemId` here if you want?

Reply:
Yeah, I think that is better. Also works better with the caching, since we don't have array in the query key.